### ISSUE TITLE
Problem when using the model for prediction

### ISSUE THREAD

**Issue by bernardohenz** [philipperemy/deep-speaker#15]
Hello,

I am having problem to use a trained model to perform prediction. I was capable of training using your code, but I'm having problems with the shape of the input to feed to the model.predict. 

Can someone help me?

**Comment by euzun** [comment]
Hello @bernardohenz,

Here is what I've tried;
`model = convolutional_model(batch_input_shape=batch_shape, batch_size=batch_size, num_frames=num_frames)`
`model.compile(optimizer='adam', loss=deep_speaker_loss)`
`model.load_weights(model_path)`
`get_last_layer_output = K.function([model.layers[0].input,K.learning_phase()],[model.get_layer('ln').output])`
        `embeddings = get_last_layer_output([x,0])[0]`

Here 'model_path' is the pretrained model weight file (.h5 file) and x is your inputs after reshaped. But, I have not get any classification results yet.

**Comment by philipperemy** [comment]
Try again with the new master. I pushed the latest code.
